### ISSUE TITLE
activation details should have duration

### ISSUE THREAD

**Issue by csantanapr** [openwhisk/openwhisk#279]
``` bash
  $ wsk activation get e315c44fb7704be0a91468ae81c23841
ok: got activation /_/e315c44fb7704be0a91468ae81c23841
```

``` json
{
    "activationId": "e315c44fb7704be0a91468ae81c23841",
    "annotations": [],
    "end": 1461869807205,
    "logs": [
        "2016-04-28T18:56:44.25689232Z  stdout: Starting a blackbox service"
    ],
    "name": "example",
    "namespace": "csantana@us.ibm.com",
    "publish": false,
    "response": {
        "result": {
            "result": {
                "msg": "Hello Rey from arbitrary C program!\n"
            }
        },
        "status": "success",
        "success": true
    },
    "start": 1461869804132,
    "subject": "csantana@us.ibm.com",
    "version": "0.0.1"
}
```

**Comment by sjfink** [comment]
I believe the duration is end - start, both of which are present above.

**Comment by mbehrendt** [comment]
1) both start and end should be in a human-readable format
2) we should include a field on duration, so users don't have to calculate it manually

**Comment by rabbah** [comment]
There's a difference between what we record and then how/what we make easier for the user to consume. This can be solved entirely in the CLI for example to mimic the UI.

**Comment by sjfink** [comment]
agreed.  It's an API not a UI.

**Comment by mbehrendt** [comment]
i think it's cumbersome for api clients having to do the math by themselves. we'll need to calculate the duration anyway server-side, for audit and billing purposes, so should expose that record via the api.

additionally, implementing it in the various clients one by one is likely going to result in inconsistencies.

**Comment by rabbah** [comment]
closed by #1471.
